components/automate-backend-elasticsidecar/habitat/plan.sh

```diff
@@ -0,0 +1,53 @@
+# shellcheck disable=SC2148
+pkg_name="automate-backend-elasticsidecar"
+pkg_description="Password manager for elasticsearch-odfe"
+pkg_origin="chef"
+vendor_origin="chef"
+pkg_maintainer="Chef Software Inc. <support@chef.io>"
+pkg_license=("Chef-MLSA")
+pkg_version="1.0.28"
+pkg_upstream_url="http://github.com/chef/a2-ha-backend/components/automate-backend-elasticsidecar"
+pkg_deps=(
+  core/bash
+  core/ruby26
+  chef/elasticsearch-odfe
+)
+pkg_build_deps=(
+  core/make
+  core/gcc
+)
+pkg_interpreters=(bin/ruby)
+
+
+# currently broken, these should be required binds, but they are never found
+pkg_binds=(
+  [elasticsearch]="http-port transport-port root-ca admin-pem admin-key admin_username admin_password dashboard_username dashboard_password"
+)
+
+pkg_lib_dirs=(lib)
+
+do_before() {
+  update_pkg_version
+}
+
+do_build() {
+  return 0
+}
+
+do_install() {
+  gem update --system
```

Comment:
There's no real benefit in doing this with modern ruby and it just makes the packages bigger

Reply:
Okay Noted, Have removed this